docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,547 @@
+
+### Launch IoT Setup Manager
+
+You can launch the IoT Setup Manager with or without using Tizen Studio:
+
+#### Launch using Tizen Studio
+
+From Tizen Studio, you can launch the IoT Setup Manager in the following two ways:
+
+-   On the menu bar, go to **IoT > IoT Setup Manager**.
+-   On the toolbar, click the **IoT Setup Manager** icon (![ISM icon](media/iot_setup_manager_icon.png)) as shown in the following image.
+
+    ![IoT Setup Manager menu path](media/tizen_studio_ism.png)
+    
+IoT Setup Manager window appears.
+
+#### Launch without using Tizen Studio
+
+During installation, the Package Manager creates shortcuts for IoT Setup Manager, which are used to launch the IoT Setup Manager as a standalone application. To launch the IoT Setup Manager follow:
+
+-   Windows, go to **Start Menu > Programs > Tizen Studio > Tools > IoT Setup Manager**.
+-   Ubuntu, go to **Dash (equivalent to Start Menu) > Applications > IoT Setup Manager**.
+
+IoT Setup Manager window appears.
+
+### The IoT Setup Manager view contains the following fields
+
+![IoT Setup Manager Main](media/setup_manager_main_window.png)
+
+-   **Select burning profile**
+    
+    You can select one of the available profiles from the **Select burning profile** drop-down list, to avoid creating or modifying profiles every time. Additionally, you can copy the flashing profile to the Manager directory. This enables the application to recognize the profile on the next start.
+
+-   **Create**
+     
+    To create a new flashing profile. For more information see, [edit profile window](#the-edit-profile-window-contains-the-following-fields)
+
+-   **Edit**
+    
+    To modify the existing profile. For more information see, [edit profile window](#the-edit-profile-window-contains-the-following-fields)
+	
+-   **Delete**
+    
+    To delete an existing profile.
+
+-   **Select Drive**
+    
+    Insert an SD card and wait until it appears in the drop-down list.
+
+-   **Burn**
+    
+    Click Burn and wait until the flashing routine is finished. This may take few minutes and you may have to enter your supervisor password, which is required to access the SD card.
+
+#### Steps to flash SD card using IoT Setup Manager
+
+To flash Tizen on your SD card using the IoT Setup Manager, follow these steps:
+
+1.   Select profile from the dropdown list.
+2.   Select SD card from the dropdown list.
+3.   Click **Burn** to burn Tizen to your SD Card.
+
+     > [!NOTE]
+     > Currently, the USB SD card reader is only supported. The internal PC SD slot is not supported.
+     > Also, if the **Select Drive** does not show anything, detach and insert the SD card again into the computer, or close and restart IoT Setup Manager.
+
+### The edit profile window contains the following fields
+
+![ISM edit profile page](media/ism_edit_rpi4_headed.png)
+
```

Comment:
Please don't use a sentence as A heading. I didn't mean it. How about below?

```suggestion
### Edit profile window fields

![ISM edit profile page](media/ism_edit_rpi4_headed.png)

The edit profile window contains the following fields

```

Reply:
Yes, this way it would be better. Thanks for the review.